lib/SILGen/SILGenTopLevel.cpp

```diff
@@ -214,7 +214,23 @@ void SILGenFunction::emitCallToMain(FuncDecl *mainFunc) {
     B.createReturn(loc, exitCode);
   } else {
     FuncDecl *exitFuncDecl = SGM.getExit();
-    assert(exitFuncDecl && "Failed to find exit function declaration");
+    if (!exitFuncDecl) {
+      // If it doesn't exist, we can conjure one up instead of crashing
+      // @_extern(c)
```

Comment:
I think this should be `@_cdecl` instead of `@_extern(c)`. I'm worried if it's `@_extern(c)` it will be included in the generated header (depending on the module that it's in).

Reply:
Sorry I was reading `@_extern` as `@_expose`. You're totally right. This is correct.